### PR TITLE
Update "Unity Development with VS Code" page…

### PR DESCRIPTION
**Pull request by eppz**
…with latest Unity (5.5) simplified integration setup.

### CODE REVIEW

File: docs/other/unity.md [hunk 1]
```diff
@@ -16,47 +16,15 @@ Visual Studio Code can be a great companion to Unity for editing and debugging C
...
+Simply open up *Unity Preferences*, *External Tools*, then browse **Visual Studio Code** executable as *External Script Editor*.
 
-1. Sets VS Code as the default editor - opening a script now opens it in VS Code
-2. Configures Unity to pass file and line numbers and reuse the existing window - so VS Code opens in the correct context
-3. Scrubs the Unity project file to ensure that OmniSharp can work with it - to get the best editing experience
-4. Configures VS Code to ignore certain Unity file types - removing clutter from the VS Code file explorer
-5. Configures a `launch.json` file with the correct debug port - to enable debugging. However, installing the [Debugger for Unity](https://marketplace.visualstudio.com/items/Unity.unity-debug) is recommended to get debugging support.
+![Unity Preferences](images/unity/Unity_Preferences_External_Script_Editor.gif)
 
-### Step 1: Download the plug-in code
```
Comment (eppz) [inline docs/other/unity.md]: The plugin is not needed anymore, as this behaviour is supported in Unity by default.   the plugin author updated the description at https://github.com/dotBunny/VSCode/blob/master/README.md#requirements

File: docs/other/unity.md [hunk 2]
```diff
@@ -16,47 +16,15 @@ Visual Studio Code can be a great companion to Unity for editing and debugging C
...
-
-To turn on the use of the provided integration, you will need to go to `Unity Preferences` and select the newly created `VSCode` tab.
-
-![VSCode Preferences](images/unity/plugin-preferences.png)
-
-Toggle on `Enable Integration` and you are ready to get going.
-
-Additionally, you can toggle `Output Messages To Console` which will echo output to the Unity console e.g. whether the debug port was found.  This is useful for debugging any issues with the integration.
-
-Clicking the `Write Workspace Settings` button will configure the workspace settings to filter out non-code assets created by Unity e.g. `.meta` files.
```
Comment (eppz) [inline docs/other/unity.md]: This is also done by Unity now.

File: docs/other/unity.md [hunk 3]
```diff
@@ -16,47 +16,15 @@ Visual Studio Code can be a great companion to Unity for editing and debugging C
...
-
-### Step 3: Open the Project
```
Comment (eppz) [inline docs/other/unity.md]: Not necessary, will be done automatically by opening up any script.

File: docs/other/unity.md [hunk 4]
```diff
@@ -79,11 +47,11 @@ Two topics that will help you are [Basic Editing](/docs/editor/codebasics.md) an
 
 ## Unity Extensions
 
-The community is continually developing more and more valuable extensions for Unity. Here are some popular extensions that you might find useful. You can find more extensions in the VS Code [Extension Marketplace](https://marketplace.visualstudio.com/VSCode).
```
Comment (eppz) [inline docs/other/unity.md]: Just modified the url to a search result page with the term "Unity".

File: docs/other/unity.md [hunk 5]
```diff
@@ -79,11 +47,11 @@ Two topics that will help you are [Basic Editing](/docs/editor/codebasics.md) an
 
 ## Unity Extensions
 
-The community is continually developing more and more valuable extensions for Unity. Here are some popular extensions that you might find useful. You can find more extensions in the VS Code [Extension Marketplace](https://marketplace.visualstudio.com/VSCode).
+The community is continually developing more and more valuable extensions for Unity. Here are some popular extensions that you might find useful. You can search for more extensions in the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/search?term=Unity&target=VSCode).
 
 <div class="marketplace-extensions-unity"></div>
 
-> Tip: The extensions shown above are dynamically queried. Click on an extension tile above to read the description and reviews to decide which extension is best for you. See more in the [Marketplace](https://marketplace.visualstudio.com/vscode).
+External Tools Tip: The extensions shown above are dynamically queried. Click on an extension tile above to read the description and reviews to decide which extension is best for you. See more in the [Marketplace](https://marketplace.visualstudio.com/vscode).
```
Comment (eppz) [inline docs/other/unity.md]: This seems added by accident. Should skip this hunk.

**Comment by gregvanl** [comment]
Thanks!

**Comment by gregvanl** [comment]
@eppz I followed up with https://github.com/Microsoft/vscode-docs/commit/22a45ce9dd9339bf9bdcb1874be38420401df49d. I wasn't completely sure how much of the Common Questions was still relevant. For example, setting command line arguments on Windows. Let me know if there is more cleanup there.

**Comment by eppz** [comment]
Sure thing!
When will this go live?

On the site - https://code.visualstudio.com/docs/other/unity - there is still the previous version.

**Comment by gregvanl** [comment]
I was waiting a bit in case I heard back from you. I was going to publish this afternoon Seattle time. We can then follow up with more cleanup later.

**Comment by eppz** [comment]
Sorry, I just missed your question. True, that section is obsolete as well. Should I commit, or you'll just remove it?

**Comment by gregvanl** [comment]
If you'd like to submit another PR, it would be helpful. I don't have Unity installed so I would be flying a bit blind.

**Comment by eppz** [comment]
Also, there may be mentioned somewhere that to have correct IntelliSense, code completion, build support one should install C# OmniSharp extension.

Not sure, but it is like VS Code suggest it automatically. If that's the case, this is not necessary.

**Comment by eppz** [comment]
Sure, I'll push that in a moment.

**Comment by eppz** [comment]
Is there a chance to sort the extensions query (at `marketplace-extensions-unity` div) by "Relevance" instead of "Downloads"? 

If that's the case, my extension would be listed in top 4, thus included in that list. 😄

**Comment by eppz** [comment]
Just double checked in Unity, and those options are no longer available (the context passed automatically).

Pull request at https://github.com/Microsoft/vscode-docs/pull/957

**Comment by gregvanl** [comment]
Sorry, we don't have enought control to the query from the client side, it's coming from the Marketplace.

**Pull request merged**